mxd/backend-service/src/main/java/org/eclipse/mxd/util/SingletonObjectMapper.java

```diff
@@ -0,0 +1,46 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *    Contributors:Ravinder Kumar
+ *    Backend-API and implementation
+ *
+ ******************************************************************************/
+
+
+
+
+package org.eclipse.mxd.util;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class SingletonObjectMapper {
+
+    private static ObjectMapper objectMapper = new ObjectMapper();
+
+    private SingletonObjectMapper() {
+    }
+
+    public static ObjectMapper getObjectMapper() {
+        if (objectMapper == null) {
+                if (objectMapper == null) {
+                    objectMapper = SingletonObjectMapper.getObjectMapper();
+                }
+        }
```

Comment:
Remove these lines, objectMapper already initialized, directly return it.

Reply:
removed unwanted lines.